datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -1,110 +1,395 @@
+                <Row>
+                    <Col xl={5} lg={5} md={5} sm={24} xs={24}>
+                        <UserSidebar>
+                            <UserSidebarSubSection>
+                                <CustomAvatar
+                                    size={160}
+                                    photoUrl={profileSrc}
+                                    name={data?.corpUser?.info?.fullName || undefined}
+                                    style={{ marginTop: '14px' }}
+                                />
+                                <UserName>{data?.corpUser?.info?.fullName || 'NA'}</UserName>
+                                <UserRole>{data?.corpUser?.info?.title || 'NA'}</UserRole>
+                                <UserTeam>{data?.corpUser?.editableProperties?.teams?.join(',') || 'NA'}</UserTeam>
+                                <Divider className="divider-infoSection" />
+                                <UserSocialDetails>
+                                    <Space>
+                                        <MailOutlined /> {data?.corpUser?.info?.email || 'NA'}
+                                    </Space>
+                                </UserSocialDetails>
+                                <UserSocialDetails>
+                                    <Space>
+                                        <SlackOutlined /> {data?.corpUser?.editableProperties?.slack || 'NA'}
+                                    </Space>
+                                </UserSocialDetails>
+                                <UserSocialDetails>
+                                    <Space>
+                                        <PhoneOutlined /> {data?.corpUser?.editableProperties?.phone || 'NA'}
+                                    </Space>
+                                </UserSocialDetails>
+                                <Divider className="divider-aboutSection" />
+                                <AboutSection>
+                                    About
+                                    <AboutSectionText>
+                                        <Paragraph
+                                            editable={{ onChange: aboutMeTextChangeFunction }}
+                                            ellipsis={{ rows: 2, expandable: true, symbol: 'Read more' }}
+                                        >
+                                            {data?.corpUser?.editableProperties?.aboutMe || 'NA'}
+                                        </Paragraph>
+                                    </AboutSectionText>
+                                </AboutSection>
+                                <Divider className="divider-groupsSection" />
+                                <GroupsSection>
+                                    Groups
+                                    <TagsSection>
+                                        {groupsDetails?.relationships.length === 0 && (
+                                            <NoDataFound>No Groups found</NoDataFound>
```

Comment:
replace `No Groups found` with `No Groups`

Reply:
use the same grey + italics that you use for "not provided" above

Reply:
Sure. Text changes added.